backend/README.md

```diff
@@ -10,9 +10,8 @@ Install Docker on your Mac.
 ```
 docker build -t backend:v1.0.0 .
 docker run -d -p 27017:27017 mongo
```

Comment:
Should this now be mern-mongo to match below line

Reply:
Yes you're right, I've added it